packages/chart/src/Chart.tsx

```diff
@@ -704,23 +708,22 @@ class Chart extends Component<ChartProps, ChartState> {
             style={{ height: '100%', width: '100%' }}
           />
         )}
-        <Popper
-          className="chart-error-popper"
-          options={{ placement: 'top' }}
-          isShown={shownError != null}
-          onExited={this.handleErrorClose}
-          closeOnBlur
-          interactive
-        >
-          {shownError != null && (
-            <>
-              <div className="chart-error">{shownError}</div>
-              <CopyButton tooltip="Copy Error" copy={shownError}>
-                Copy Error
-              </CopyButton>
-            </>
-          )}
-        </Popper>
+        {downsamplingError != null && (
```

Comment:
wasn't quite sure what the difference is between downsampling errors and other types of errors. Do we not want them to show in this window as well?

Currently I have them trigger their own overlay warning